content/tutorials/r_setup_codingclub_session/index.Rmd

```diff
@@ -17,12 +17,12 @@ knitr::opts_chunk$set(eval = FALSE)
 ```
 
 
-Before each coding club you will usually have to download some data and some scripts.
-This can be done manually by going to each of the GitHub pages but it is much easier to use the `setup_codingclub_session()` function of the `inborutils` package.
+Before each coding club you will usually have to download some datasets and some scripts.
+This can be done manually from the correspondent GitHub webpages but it is much easier to use the `setup_codingclub_session()` function of the `inborutils` package.
```

Comment:
```suggestion
This can be done manually from the corresponding GitHub webpages but it is much easier to use the `setup_codingclub_session()` function of the `inborutils` package.
```